lib/decision_review_v1/configuration.rb

```diff
@@ -8,7 +8,9 @@ module DecisionReviewV1
   # sets the base path, the base request headers, and a service name for breakers and metrics.
   #
   class Configuration < Common::Client::Configuration::REST
-    self.read_timeout = Settings.caseflow.timeout || 20 # using the same timeout as lighthouse
+    # add some padding to timeout to give Lighthouse more time to return 504 JSON
+    self.read_timeout = Settings.caseflow.timeout ? (Settings.caseflow.timeout + 15) : 20
```

Comment:
@data-doge i'd love to hear your thoughts on this before i clean up the commented out code below

Reply:
Thanks for looking into this @anniebtran! Two thoughts: 
1. Lighthouse, pointing to [a DD trace](https://vagov.ddog-gov.com/apm/trace/700740639041129895?colorBy=service&env=eks-prod&graphType=flamegraph&shouldShowLegend=true&sort=time&spanID=1438754843897795079&spanViewType=metadata&timeHint=1696358115949.193&traceSearchQuery=%7B%22traceQuery%22%3A%22%22%2C%22spanQueries%22%3A%7B%22a%22%3A%22%22%7D%7D&view=spans), believed that our timeout was only 1 minute. As you point out here, the caseflow timeout is 119 seconds. Does that mean we need to modify another timeout configuration somewhere?
2. Do we also need to update the read_timeout in `lib/decision_review/configuration.rb`?

Reply:
Great questions @data-doge!

1. Ugh I'm annoyed I can't go in and look at the trace yet 😂 but yeah I wasn't sure where they were getting 1 min from or if there was some truncation happening because it's configured at 1min 59sec. I can try to poke around deeper to see if there are other configs that could be overwriting it, but I feel like I'm kind of throwing things at the wall and seeing what might stick while I'm unable to see what they were looking at while they were investigating. 😩 
2. I think that config is for the v0 controllers — are we still using those? If so, I can totally update it there, I just didn't know if we were still using them 😅 Good catch!